core/system/src/svc.c

```diff
@@ -233,7 +245,8 @@ void UVISOR_NAKED SVCall_IRQn_Handler(void)
 
         :: [svc_mode_mask]       "I" ((UVISOR_SVC_MODE_MASK) & 0xFF),
            [svc_fast_index_mask] "I" ((UVISOR_SVC_FAST_INDEX_MASK) & 0xFF),
-           [svc_vtor_tbl_count]  "i" (UVISOR_ARRAY_COUNT(g_svc_vtor_tbl) - 1)
+           [svc_vtor_tbl_count]  "i" (UVISOR_ARRAY_COUNT(g_svc_vtor_tbl) - 1),
+           [priv_svc_0]          "i" (offsetof(UvisorPrivSystemIRQHooks, priv_svc_0))
```

Comment:
@Patater : Please use the address of priv_svc_0 inside the global variable to avoid the priv_svc_0-offset calculation.